pkg/cmd/condition/delete.go

```diff
@@ -63,24 +63,51 @@ or
 				return err
 			}
 
-			return deleteConditions(s, p, args)
+			return deleteConditions(s, p, args, opts.DeleteAllNs)
 		},
 	}
 	f.AddFlags(c)
 	c.Flags().BoolVarP(&opts.ForceDelete, "force", "f", false, "Whether to force deletion (default: false)")
+	c.Flags().BoolVarP(&opts.DeleteAllNs, "all", "", false, "Delete all Conditions in a namespace (default: false)")
 	_ = c.MarkZshCompPositionalArgumentCustom(1, "__tkn_get_condition")
 	return c
 }
 
-func deleteConditions(s *cli.Stream, p cli.Params, condNames []string) error {
+func deleteConditions(s *cli.Stream, p cli.Params, condNames []string, deleteAll bool) error {
 	cs, err := p.Clients()
 	if err != nil {
 		return fmt.Errorf("failed to create tekton client")
 	}
 	d := deleter.New("Condition", func(conditionName string) error {
 		return cs.Tekton.TektonV1alpha1().Conditions(p.Namespace()).Delete(conditionName, &metav1.DeleteOptions{})
 	})
+	if deleteAll {
+		condNames, err = allConditionNames(p, cs)
+		if err != nil {
+			return err
+		}
+	}
 	d.Delete(s, condNames)
-	d.PrintSuccesses(s)
+
+	if !deleteAll {
+		d.PrintSuccesses(s)
+	} else if deleteAll {
```

Comment:
i think the logic can be a bit more clearer, ie : 


if deleteall { 
....
} else {
}

Reply:
The linter complains if I do it with else. It says to be more explicit.

Reply:
that's so weird....... did you take a note of which checker from golangci-lint this is coming from?

Reply:
No, but we can try it and get the results in this pr if you like?

Reply:
Yeah, so it fails with the following:

```
I0210 18:05:31.332] ==== TESTING IF GOLINT HAS BEEN DONE ====
I0210 18:05:31.333] =========================================
I0210 18:07:53.240] pkg/cmd/condition/delete.go:94:9: elseif: can replace 'else {if cond {}}' with 'else if cond {}' (gocritic)
I0210 18:07:53.246] 	} else {
I0210 18:07:53.246] 	       ^
```

Reply:
fair enough, thanks for tracking this. I'll try to do a bit more diging when i have some time because this doesn't feel right.